hooks/useSocialLogo.ts

```diff
@@ -6,7 +6,16 @@ const useSocialLogo = (session: Session | null) => {
     return "google";
   } else if (id?.includes("DISCORD")) {
     return "discord";
-  } else return null;
+  } else if (id?.includes("TWITTER")) {
+    return "twitter";
+  } else if (id?.includes("TWITCH")) {
+    return "twitch";
+  } else if (id?.includes("REDDIT")) {
+    return "reddit";
+  } else if (id?.includes("CHESS")) {
+    return "chess";
+  } 
```

Comment:
In the future, please modify the frontend app to support the same Id's from the backend. This way we don't need to create and maintain functions like this, which need to be changed whenever a new type is added.